packages/e2e-tests/src/specs/editor/editor.js

```diff
@@ -65,24 +66,4 @@ describe('Story Editor', () => {
     expect(crossOriginIsolated).toBeFalse();
     await toggleVideoOptimization();
   });
-
-  //eslint-disable-next-line jest/no-disabled-tests -- TODO(#12023): Fix flakey test.
-  it.skip('should preview story with development mode', async () => {
-    await createNewStory();
-
-    const editorPage = page;
-    const previewPage = await previewStory(editorPage);
-
-    await previewPage.waitForSelector('.i-amphtml-story-dev-tools-header');
-
-    await expect(previewPage).toMatch(/Preview/i);
-    await expect(previewPage).toMatch(/Debug/i);
-    await expect(previewPage).toMatch(/Add device/i);
-
-    await previewPage.waitForSelector('amp-story-player');
-    await expect(previewPage).toMatchElement('amp-story-player');
-
-    await editorPage.bringToFront();
-    await previewPage.close();
```

Comment:
Removed without replacing?

Reply:
Yes, this is already covered by every other test involving preview.